index.html

```diff
@@ -5338,7 +5341,7 @@ <h3 id="Scoundrel">Scoundrel</h3>
                      move within 5ft of during this movement only. The same creature cannot be hit
                      twice.
                </td>
-               <td>A+E, 5ft, Multiple Target
+               <td>A+E, 5ft, Multiple target
```

Comment:
surely this should be plural?  Multiple targets?

Reply:
Tbf that's the only use of the term Multiple Target, so it shouldn't be there at all. I'll change to "All within 5ft of 75ft movement" like Rampage is worded